saleor/graphql/menu/mutations.py

```diff
@@ -196,6 +198,103 @@ def user_is_allowed(cls, user, input):
         return user.has_perm('menu.manage_menus')
 
 
+_MenuMoveOperation = namedtuple(
+    '_MenuMoveOperation', ('menu_item', 'parent', 'sort_order'))
+
+
+class MenuItemMove(BaseMutation):
+    menu_item = graphene.List(
```

Comment:
I think we would make @dominik-zeglen's life easier, if we also returned the whole `menu` that this item belongs to, is that right Dominik? Apollo Client would be able to refresh the whole menu in its cache based on that.

Reply:
Let me know.

Reply:
Definitely!